frontend/src/component/addons/AddonForm/AddonForm.tsx

```diff
@@ -334,7 +334,7 @@ export const AddonForm: VFC<IAddonFormProps> = ({
                             type="submit"
                             color="primary"
                             variant="contained"
-                            permission={ADMIN}
+                            permission={editMode ? 'UPDATE_ADDON' : 'CREATE_ADDON'}
```

Comment:
You can switch the import to grab the `CREATE_ADDON` and `UPDATE_ADDON` constants instead, so we don't have to use string literals: 
![image](https://user-images.githubusercontent.com/14320932/196437796-87e2f721-c897-4891-ab4c-e181bc9f76c0.png)

Reply:
will do!